spec/features/locales_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe 'locale switching' do
+
+  context 'when arabic is selected' do 
```

Comment:
Then, I will have to remove the `context` of this test as you mentioned before that it should always have some 'stuff' such as 'let / def' or 'before / after'. Correct?